Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -143,14 +143,24 @@ - (void)updateAuthorConversationColorNameWithTransaction:(YapDatabaseReadTransac
 {
     OWSAssertDebug(transaction);
 
-    if (self.interaction.interactionType != OWSInteractionType_IncomingMessage) {
-        _authorConversationColorName = nil;
-        return;
+    switch (self.interaction.interactionType) {
+        case OWSInteractionType_TypingIndicator: {
+            OWSTypingIndicatorInteraction *typingIndicator = (OWSTypingIndicatorInteraction *)self.interaction;
+            _authorConversationColorName =
+                [TSContactThread conversationColorNameForRecipientId:typingIndicator.recipientId
```

Comment:
Hmm... why do we need ` _authorConversationColorName` for a typing indicator?

Reply:
Because in a group, we show the avatar of the "typing" user next to the typing indicator and we want to honor their conversation color in the default avatar icon.